generators/chipyard/src/main/scala/config/RocketConfigs.scala

```diff
@@ -21,6 +21,19 @@ class TinyRocketConfig extends Config(
   new freechips.rocketchip.subsystem.With1TinyCore ++             // single tiny rocket-core
   new chipyard.config.AbstractConfig)
 
+class MempressRocketBaseConfig extends Config(
+  new freechips.rocketchip.subsystem.WithInclusiveCache(nBanks=8, nWays=16, capacityKB=2048) ++
+  new WithExtMemIdBits(7) ++
```

Comment:
Where is this defined?

Reply:
Can you still give the full package path to the config fragment? I.e. `chipyard.config.With...`